pkg/asset/tls/certkey.go

```diff
@@ -69,6 +70,39 @@ func (c *CertKey) Load(asset.FileFetcher) (bool, error) {
 	return false, nil
 }
 
+func (c *CertKey) loadCertKey(f asset.FileFetcher, filenameBase string) (bool, error) {
+	if os.Getenv("OPENSHIFT_INSTALL_LOAD_CLUSTER_CERTS") != "true" {
```

Comment:
Who is setting this? Will it be set by the agent installer or users? We have avoided using any user-set environment variables.

Reply:
It'll be set by the assisted-installer. This feature is not intended for consumption by users running openshift-install directly, hence the env var check.

Reply:
Similar to the invoker one:
https://github.com/openshift/installer/blob/6d778f911e79afad8ba2ff4301eda5b5cf4d8e9e/pkg/asset/openshiftinstall/openshiftinstall.go#L78

Reply:
Avoiding env vars set by users is my main concern. This looks reasonable as the env var is avoiding this functionality creeping outside of the agent installer.

/approve